pkg/ctl/delete/cluster.go

```diff
@@ -103,11 +106,27 @@ func doDeleteCluster(rc *cmdutils.ResourceCmd) error {
 	}
 
 	{
+
+		logger.Info("cleaning up LoadBalancer services")
+		if err := ctl.GetCredentials(cfg); err != nil {
+			return err
+		}
+		cs, err := ctl.NewStdClientSet(cfg)
+		if err != nil {
+			return err
+		}
+		ctx, cleanup := context.WithTimeout(context.Background(), 10*time.Minute)
```

Comment:
We have a global `--timeout` flag, it's defined as 'wait time in any polling operations (default 25m0s)'.
I think we should use it here, but it would a good idea to review how we do polling in general, as what's here is different from other places. I don't think we have to address it now, I'll open an issue.

Reply:
I've opened #1036.